tasks/list-remaining-migrated-removals.ts

```diff
@@ -0,0 +1,207 @@
+import { readFileSync, writeFileSync } from 'fs';
+
+import { task, types } from 'hardhat/config';
+import chalk from 'chalk';
+import { ethers } from 'ethers';
+
+import type { FireblocksSigner } from '../plugins/fireblocks/fireblocks-signer';
+
+interface ListMigratedRemovalsTaskOptions {
+  file: string;
+  outputFile?: string;
+  dryRun?: boolean;
+}
+
+type ParsedListMigratedRemovalsTaskOptions = RequiredKeys<
+  ListMigratedRemovalsTaskOptions,
+  'file'
+>;
+
+export const GET_LIST_MIGRATED_REMOVALS_TASK = () =>
+  ({
+    name: 'list-remaining-migrated-removals',
+    description:
+      'Utility to list migrated removals for sale that were not included in migrated certificates',
+    run: async (
+      options: ListMigratedRemovalsTaskOptions,
+      _: CustomHardHatRuntimeEnvironment
+    ): Promise<void> => {
+      const {
+        file,
+        outputFile = 'listed-migrated-removals.json',
+        dryRun,
+      } = options as ParsedListMigratedRemovalsTaskOptions;
+      const jsonData = JSON.parse(readFileSync(file, 'utf8'));
+      // hre.log({ jsonData });
+
+      const [signer] = await hre.getSigners();
+      const signerAddress = await signer.getAddress();
```

Comment:
wonder if we should do validation on network and address (e.g., to verify the address used is the consignor, on a valid network, and uses fireblocks depending on the network)

Reply:
so I'm adding validation to make sure the signer has the `CONSIGNOR` role and that the network is one of `localhost`, `mumbai`, or `polygon` .... I'm not sure how to handle checking that the signer is a fireblocks signer.  maybe you just mean that based on the network we should cast the signer as a `FireblocksSigner` ? I guess we could then check that the signer has one of the fireblocks-only properties?

Reply:
there should be fireblocks-exclusive properties on the signer object, e.g [this one](https://github.com/nori-dot-eco/contracts/blob/master/plugins/fireblocks/fireblocks-signer.ts#L35)

Reply:
Have we thought about which Fireblocks wallet should function as the consignor?  Can it be the contract owner or should we set up a new wallet address to function as the consignor?